micro-ui/web/micro-ui-internals/packages/modules/workbench/src/configs/UICustomizations.js

```
@@ -1003,4 +1003,11 @@ export const UICustomizations = {
       },
     },
   },
+  MySidebarDataSearchConfig: {
+    preProcess: (data, additionalDetails) => {
+      data.body.MdmsCriteria.filters.displayName = data?.state?.searchForm?.displayName;
+      delete data.displayName;      
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)